### DISCUSSION TITLE
Matrix Space/Room

### DISCUSSION THREAD

**Discussion by trymeouteh** [tauri-apps/tauri#3968] [Ideas]
Please create an official matrix space or room to discuss and ask questions on Tauri. Discord has a horrible privacy policy and is not open source unlike Matrix.

**Comment by FabianLars** [comment]
We're not completely opposed to the idea and thought about having a discord-matrix bridge in the past, but in the end, the interest for a matrix room was way too low to justify the additional "overhead" for the team.

This may be something we'll look into a while after 1.0-stable was release (so it settled down a bit), _if_ there's enough interest (both in the community and in our team to moderate it/be active in there).
